### PR TITLE
[KYC-89] Docker composition

### PR DESCRIPTION
**Pull request by yreynhout-lykke**
The aim is to ease spinning up the UI and verify that everything works without too many manual steps.

- Created a docker composition to host postgres, ironclad, mailhog and apollo together 
- as well as a script to initialize ironclad.

The `docker-compose.yml` does require a few environment variables to work properly. Specifying `DB__DATACONNSTRING, APPINSIGHTSKEY, ONFIDO__APIKEY, ONFIDO__URI, TWILIO__ACCOUNTSID, TWILIO__AUTHTOKEN, TWILIO__CALLVERIFICATIONFLOWSID, TWILIO__FROMPHONENUMBER, TWILIO__STATUSAPIURL` in an `.env` file should do the trick. We may _in the future_ want to look at how we could reuse one of our azure key vaults. Right now, these either contain production values or values for the automated testing to work.

The passwords and secrets in plain text in this PR are harmless since they are merely meant to make sure the docker composed services work together.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      - "25432:5432"
+  ironclad:
+    image: lykkecorp.azurecr.io/ironclad:latest
+    depends_on:
+      - postgres
+    environment:
+      SERVER__DATABASE: 'Host=postgres;Database=ironclad;Username=postgres;Password=postgres;'
+      SERVER__ISSUER_URI: 'http://ironclad.io'
+      SERVER__RESPECT_X_FORWARDED_FOR_HEADERS: 'True'
+      API__AUTHORITY: 'http://localhost:5005'
```
Comment (jchannon) [inline docker-compose.yml]: Joe has added an endpoint in Apollo to return this info. Not sure if that helps or not here

Comment (yreynhout-lykke) [reply docker-compose.yml]: Doubt it, no http invocation at this stage. Purely compositional configuration going on.

File: run-local.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+done
+printf '\n'
+
+if [ ! -f .tools/ironclad ]; then
+    printf 'Installing Ironclad CLI in .tools'
+    dotnet tool install ironclad --tool-path=.tools --version=0.6.0-beta.2.28
+fi
+.tools/ironclad login http://localhost:5005
+.tools/ironclad scopes add "api" -u "phone_number" -u "phone_number_verified" -u "email" -u "email_verified" -u "given_name" -u "family_name" -u "birthdate" -u "country" -u "street" -u "postcode"
+.tools/ironclad apis add "api" "secret" -a "apollo_api" -c "phone_number" -c "phone_number_verified" -c "email" -c "email_verified" -c "given_name" -c "family_name" -c "birthdate" -c "country" -c "street" -c "postcode"
```
Comment (JoeStead) [inline run-local.sh]: It'd be nice to use the same values we set in in environment variables for docker-compose as here, I suspect this isn't possible though.

Comment (yreynhout-lykke) [reply run-local.sh]: Find me a way and I'll implement it for you ...

Comment (JoeStead) [reply run-local.sh]: :)

File: .gitignore [hunk 3]
```diff
@@ -84,3 +84,5 @@ node_modules
 *.njsproj
 *.sln
 *.sw*
+.env
+.tools
```
Comment (JoeStead) [inline .gitignore]: good catch

**Pull request merged**